src/NUnitEngine/mock-assembly-v2/MockAssembly.cs

```diff
@@ -0,0 +1,272 @@
+// ***********************************************************************
+// Copyright (c) Charlie Poole and TestCentric GUI contributors.
```

Comment:
Should this be changed? I found a similar header in `AgentProcess.cs` and `AgentProcessTests.cs`

Reply:
Will change these in next commit.